Articles/Blog/RseStoriesPodcast.md

```diff
@@ -10,75 +10,73 @@
-engineers that I knew about, and I created a video:
-
-<br>
+encourage folks to use it. They largely didn't. I had also tried encouraging written
+stories, but people just didn't have the time to sit down and write a small article.
+I had also spent a few weeks writing down all the different kinds of research software
+engineers that I knew about, and created a video:
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/trAfA9VWLTQ" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-<br>
-
-That still reflected my own narrow life experience — I could never imagine the stories
-that I'm just not familiar with. It wasn't good enough. 
-I needed a mechanism that allows the research software engineers to tell their own stories without 
-requiring much of their time. And I wanted a format in which the stories would be easy to digest, not overewhelming
-us with yet more reading material. Then the answer hit me — and it was such a simple idea. 
-Whether we go for a run
-or embark on a daily commute, the podcast is  music
-for our ears. And so I decided — without much thought about
-the immense amount of work it takes to devise, manage, and maintain a podcast, 
-schedule people, edit audio, and edit it again — that I would create such a podcast.
+That still reflected my narrow life experience - I could never imagine the stories
+that I'm just not familiar with. It wasn't good enough. How could I tell all of these real stories?
+It hit me like a pillow falling from the ceiling - and it was such a simple idea! 
+The research software engineers needed to tell their own stories, and in a format
+that wouldn't ask much of their time. In parallel the stories needed to be easy to digest -
+many of us are overwhelmed with reading material, but what about when we go for runs?
+Or embark on a daily commute? This is when the medium of a podcast is lovely music
+for our ears. I decided in a split second moment, and without much thought about
+the immense amount of work it takes to create, manage, and maintain a podcast, 
+schedule people, edit audio (and edit it again), that I needed to create a podcast.
 
 ### How do we do something we've never done before?
 
 When in doubt, ask the internet, and look to the tools that you already know and use.
 I figured that I could post the podcast as a static site on [GitHub](https://github.com/usrse/rse-stories), 
 deployed via GitHub pages to render at [https://us-rse.org/rse-stories/](https://us-rse.org/rse-stories/).
-I needed to do a small amount of searching to figure out what kind of feed was needed for itunes
-and then how to register it. For recording technology, I went to the tools that I had
-and wound up using a Zoom meeting to record the audio and  [Audacity](https://www.audacityteam.org/) to edit it.
-And that largely was it! I follow the guideline "Do one's best and stay away from striving
```

Comment:
It's strange this was changed to past tense, because this is where it actually becomes present tense :)